### PR TITLE
[Docs] add notes for download kinetics

### PR DESCRIPTION
**Pull request by congee524**

### CODE REVIEW

File: tools/data/kinetics/README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ For better decoding speed, you can resize the original videos into smaller sized
 python ../resize_videos.py ../../../data/${DATASET}/videos_train/ ../../../data/${DATASET}/videos_train_256p_dense_cache --dense --level 2
 ```
 
-You can also download from [Academic Torrents](https://academictorrents.com/)([kinetics400](https://academictorrents.com/details/184d11318372f70018cf9a72ef867e2fb9ce1d26) & [kinetics700](https://academictorrents.com/details/49f203189fb69ae96fb40a6d0e129949e1dfec98) are avaiable) and [cvdfoundation/kinetics-dataset](https://github.com/cvdfoundation/kinetics-dataset)(Host by Common Visual Data Foundation and Kinetics400/Kinetics600/Kinetics-700-2020 are available)
+You can also download the video datasets with short edge 256 (256p) from [Academic Torrents](https://academictorrents.com/)([kinetics400](https://academictorrents.com/details/184d11318372f70018cf9a72ef867e2fb9ce1d26) & [kinetics700](https://academictorrents.com/details/49f203189fb69ae96fb40a6d0e129949e1dfec98) are avaiable) and [cvdfoundation/kinetics-dataset](https://github.com/cvdfoundation/kinetics-dataset)(Host by Common Visual Data Foundation and Kinetics400/Kinetics600/Kinetics-700-2020 are available)
```
Comment (innerlee) [inline tools/data/kinetics/README.md:66]: Do Academic Torrents and cvdfoundation both host 256p?

**Review by innerlee — APPROVED** [review]

File: tools/data/kinetics/README.md (hunk 1 above)
Comment (irvingzhang0512) [reply tools/data/kinetics/README.md:66]: Checked a few videos from cvdfoundation and they are not resize to shortside 256

Comment (irvingzhang0512) [reply tools/data/kinetics/README.md:66]: maybe shortside 360

Comment (congee524) [reply tools/data/kinetics/README.md:66]: hey, I have double-checked the videos in cvdfoundation and found some videos with short edge 720, maybe they are not resized to a fixed length.

**Review by innerlee — APPROVED** [review]

**Pull request merged**